### DISCUSSION TITLE
Q: How do I load a dataset using the NPM Package

### DISCUSSION THREAD

**Discussion by ghost** [Ironclad/rivet#440] [Q&A]
How do I load a dataset and use it on the project using: @ironclad/rivet-node

**Comment by remixer-dec** [comment]
This approach seems to be working https://github.com/ai-made-approachable/rivet-chat-api/blob/57c5fc003c76f1ea105dbd92a7b7b247e4cec51c/src/graphManager.ts#L73-L79
